cli/cli.hs

```diff
@@ -163,6 +165,7 @@ data Command w
       , loopDetectionHeuristic :: w ::: LoopHeuristic   <!> "StackBased" <?> "Which heuristic should be used to determine if we are in a loop: StackBased (default) or Naive"
       , abstractArithmetic    :: w ::: Bool             <?> "Use abstraction-refinement for complicated arithmetic functions such as MulMod. This runs the solver first with abstraction turned on, and if it returns a potential counterexample, the counterexample is refined to make sure it is a counterexample for the actual (not the abstracted) problem"
       , abstractMemory    :: w ::: Bool                      <?> "Use abstraction-refinement for Memory. This runs the solver first with abstraction turned on, and if it returns a potential counterexample, the counterexample is refined to make sure it is a counterexample for the actual (not the abstracted) problem"
+      , nodecompose   :: w ::: Bool                     <?> "Don't decompose storage slots into separate arrays"
```

Comment:
Can we change it to `noDecompose`? The cli switch will be `--no-decompose` which I feel reads better and is similar to other CLI tools https://clig.dev/#arguments-and-flags. At the moment it could be also understood as "node compose".

Reply:
Sure, sorry, good point. Fixed!